tools/ete/ete_macros.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <macros>
-    <token name="@VERSION@">3.0.0b35</token>
+    <token name="@VERSION@">3.0.0b35.1</token>
```

Comment:
This is used also for the `requirement`, please revert and modify only the `version` for the modified `tool`.

Reply:
fixed